### PR TITLE
feat(github): add code of conduct, pull request template, replace issue templates with yml version

### LINKED ISSUE #500: Add a Pull Request Template
**Is your feature request related to a problem? Please describe.**
The lack of a Pull Request Template disrupts the project.

**Describe the solution you'd like**
At least add a checkbox for pre-commit because too many people don't run pre-commit.

**Additional context**

### PR DESCRIPTION
**Pull request by 34j**
Closes #500

- Changes the Issue Template to YML format to increase the probability that it will be written in more detail.
- Adds a Pull Request Template and an empty Code of Conduct.

Since I am not a native speaker, I would be happy to be corrected if there is any weird English. Thank you very much in advance.

### CODE REVIEW

**Comment by browniebroke** [comment]
That looks really useful, I'll take a look in a bit. Do you have an example repo where you're already using that?

**Comment by 34j** [comment]
This repo was created with a branch that merged all my PRs. https://github.com/34j/vr180-convert

File: project/.github/PULL_REQUEST_TEMPLATE.md.jinja [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<!--
+  😀 Wonderful!  Thank you for opening a pull request.
+
+  By submitting this issue, you agree to follow our [Code of Conduct](https://github.com/{{ github_username }}/{{ project_slug }}/blob/main/CODE_OF_CONDUCT.md).
```
Comment (browniebroke) [inline project/.github/PULL_REQUEST_TEMPLATE.md.jinja:4]:
```suggestion
  By submitting this pull request, you agree to follow our [Code of Conduct](https://github.com/{{ github_username }}/{{ project_slug }}/blob/main/.github/CODE_OF_CONDUCT.md).
```

File: project/.github/PULL_REQUEST_TEMPLATE.md.jinja [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+<!--
+  Please make sure to review and check all of the following to merge this PR.
+
+  Note that there is no problem if they are not checked when this PR is created.
+
+  If an item is not applicable, you can add "N/A" to the end.
+-->
+
+- [ ] Code is up-to-date with the `main` branch
+- [ ] This pull request follows [Contributing.md](https://github.com/{{ github_username }}/{{ project_slug }}/blob/main/CONTRIBUTING.md)
```
Comment (browniebroke) [inline project/.github/PULL_REQUEST_TEMPLATE.md.jinja:37]:
```suggestion
- [ ] This pull request follows the [contributing guidelines](https://github.com/{{ github_username }}/{{ project_slug }}/blob/main/CONTRIBUTING.md).
```

**Review by browniebroke — APPROVED** [review]
Looks very good. I noted that the CoC link are going to the root of the repo but it's actually under .github

**Comment by browniebroke** [comment]
Thanks a lot for these 🎉

**Pull request merged**